website/docs/reference/dbt_project.yml.md

```diff
@@ -1,6 +1,8 @@
 
 Every [dbt project](/docs/build/projects) needs a `dbt_project.yml` file — this is how dbt knows a directory is a dbt project. It also contains important information that tells dbt how to operate your project.
 
+dbt uses YAML in a few different places. If you're new to YAML, it would be worth taking the time to learn how arrays, dictionaries, and strings are represented.
```

Comment:
Is there a link we can send people to for learning YAML?? I found these: https://www.redhat.com/sysadmin/yaml-beginners or https://www.w3schools.io/file/yaml-introduction/